.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateExtractor.cs

```diff
@@ -93,6 +93,30 @@ public List<Token> ExtractRelativeDurationDateWithInPrefix(string text, List<Ext
                         }
                     }
                 }
+                else if (Config.CheckBothBeforeAfter)
+                {
+                    // Check also afterStr
+                    match = Config.InConnectorRegex.MatchBegin(afterStr, trim: true);

```

Comment:
Instead of basically repeating the code in the 'else', could you turn the bodies into a method? 
Ex: call the method with the default direction to check, else if Config.CheckBothBeforeAfter, call the same method with the other direction. 
I see the same in other parts of the code. This would greatly ease maintenance in the future.

Reply:
Done

Reply:
Note, the file had an inconsistent line ending, it has been changed.